### PR TITLE
antool: dup* is supported

### PR DESCRIPTION
**Pull request by mslusarz**
This change is [Reviewable](https://reviewable.io/reviews/pmem/pmemfile/343)

### CODE REVIEW

File: src/tools/antool/listsyscalls.py [hunk 1]
```diff
@@ -412,7 +412,7 @@ def is_supported(self, syscall):
                 return RESULT_UNSUPPORTED_AT_ALL
 
             # the following syscalls are not supported YET
-            if syscall.name in ("dup", "dup2", "dup3", "flock"):
```
Comment (GBuella) [inline src/tools/antool/listsyscalls.py]: we don't have a dup3, YET.

**Comment by krzycz** [comment]
Reviewed 1 of 1 files at r1.
Review status: all files reviewed at latest revision, all discussions resolved.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/343)*

File: src/tools/antool/listsyscalls.py (hunk 1 above)
Comment (mslusarz) [reply src/tools/antool/listsyscalls.py]: LOL. So now we should revert part of PR#338 (which you approved, BTW)... Or are you going to implement dup3 soon?

Comment (GBuella) [reply src/tools/antool/listsyscalls.py]: Well, I guess I can implement dup3

**Comment by sarahjelinek** [comment]
:lgtm:

---

Review status: :shipit: all files reviewed at latest revision, all discussions resolved, all commit checks successful.

---



*Comments from [Reviewable](https://reviewable.io:443/reviews/pmem/pmemfile/343#-:-Ktm4zlriAk1oVSKudbB:bnfp4nl)*

**Pull request merged**